src/modules/app/StateType.js

```diff
@@ -47,6 +48,35 @@ export type CategoryRouteStateType = {|
   +code: ErrorCodeType
 |}
 
+export type PoiRouteConfigType = {|
+  +path: ?string, // path is null for the poi-lists route
+  +language: string,
+  +city: string
+|}
+
+type LanguageKey = string
+type Path = ?string // Path can be falsy if the current displayed view is a list of events or POIs
+type AllAvailableLanguagesType = $ReadOnlyMap<LanguageKey, Path>
```

Comment:
I'd rather inline `Path` and `LanguageKey`. Not sure if a type for `AllAvailableLanguageType` is already declared somewehre else. If not, maybe better to just inline it completely for consistency.

Reply:
I suggested to do it this way because it was very unclear why Path can be optional. therefore that comment was added